src/utils/services/TodoCrud.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+import {BASE_URL} from "../constants";
+
+const getTodoList = () => {
+    return new Promise((resolve, reject) =>{
+        axios.get(BASE_URL)
+            .then((res)=>{
+                resolve(res.data.slice(0,3));
+            })
+            .catch((reason) =>{
+                reject(reason)
+            })
+    });
```

Comment:
It appears that we are using a promise here just for the `res.data.slice(0,3)`, we can move this to where `geTodoList` is being used and return Promise from axios directly.

Reply:
It might be used throughout the app, so by using this way we can handle network related errors, mapping data etc in a single place. Thats what I had in my mind while developing this.